opentelemetry-proto/Cargo.toml

```diff
@@ -29,7 +29,7 @@ path = "tests/json_deserialize.rs"
 
 
 [features]
-default = ["full"]
+populate-logs-event-name = []
```

Comment:
why change default?

Reply:
There is no feature in this crate. So `default` had no effect. And with feature `populate-logs-event-name`. we want this to be switched off by default.